muspinsim/muspinsim.xml

```diff
@@ -25,9 +25,9 @@
         <organization url="https://muon-spectroscopy-computational-project.github.io/index.html" name="The Muon Spectroscopy Computational Project"/>
     </creator>
     <requirements>
+        <requirement type="package" version="1.1.0">muspinsim</requirement>
```

Comment:
```suggestion
        <requirement type="package" version="@TOOL_VERSION@">muspinsim</requirement>
```
(You can update the macro tokens if you like, but it's not strictly needed right now.)